largest-series-product/series_product.t

```diff
@@ -6,22 +6,15 @@ use Test::Exception;
 
 my $module = $ENV{EXERCISM} ? 'Example' : 'Series';
 
-plan tests => 13;
+plan tests => 14;
```

Comment:
this is me noticing that I forgot to update that in #106 , so I had to make up for it.